proposed/extended-coding-style-guide.md

```diff
@@ -122,22 +125,77 @@ declaration.
 
 There MUST be one `use` keyword per declaration.
 
-There MUST be one blank line after the `use` block.
+When using multiple classes, functions or constants within one namespace, you
+MUST group use statements within one namespace.
+
+Use statements MUST be in a block according to the entity (class, function or group)
+which is being grouped. Within each block there MUST be no blank lines. If a block
+has multiple lines there MUST be a blank line before the first line and a blank line
+after the last line.
+
+Classes, functions or constants grouped together into a single line must be listed
+alphabetically.
+
+The groups MUST be ordered such that classes are first, followed by functions and
+then constants.
 
 For example:
 
 ```php
 <?php
+declare(strict_types=1);
+
 namespace Vendor\Package;
 
-use FooClass;
-use BarClass as Bar;
-use OtherVendor\OtherPackage\BazClass;
+use Vendor\Package\{ClassA as A, ClassB, ClassC as C};
+use Vendor\Package\Namespace\ClassD as D;
+use Vendor\Package\AnotherNamespace\ClassE as E;
+
+use function Vendor\Package\{func_a, func_b, func_c};
+use const Vendor\Package\{ConstantA, ConstantB, ConstantC};
+
+class FooBar
+{
+    // ... additional PHP code ...
+}
+
+```
+
+All files MUST declare strict types.
+
+Files containing only PHP MUST contain the strict type declarations on the
+first line preceeding the opening PHP tag.
+
+There MUST not be a blank line before the strict types declaration.
+
+For example:
+
+```php
+<?php
+declare(strict_types=1);
+
+namespace Vendor\Package;
 
 // ... additional PHP code ...
 
 ```
 
+Files containing HTML outside PHP opening and closing tags MUST on the first
```

Comment:
Is the wording correct? Shouldn't it be "MUST include an opening tag on the first line"?